app/Console/Kernel.php

```diff
@@ -100,7 +100,9 @@ protected function schedule(Schedule $schedule)
 
         $schedule->command(LoadExchanges::class)->daily();
 
-        $schedule->command(FetchExchangesDetails::class)->hourly();
+        if (config('arkscan.network') === 'production') {
```

Comment:
maybe base it off the `can_be_exchanged` attribute instead?

Reply:
hmm smart